### DISCUSSION TITLE
In search, i would like to filter by latest.

### DISCUSSION THREAD

**Discussion by markdesign** [dailydotdev/daily#1458] [Feature Requests]
When I search for a topic, I get a lot of resulst that are outdated. It would be helpful to sort by latest.

**Comment by idoshamun** [comment]
Hi @markdesign,

Thanks a lot for the suggestion. We will consider it

**Comment by sabdailydev** [comment]
@markdesign Good news - we have added a few filters to search results .  Screenshots below. Feel free to try and let us know what you think. 
<img width="572" height="381" alt="Screenshot 2025-07-14 at 10 09 55" src="https://github.com/user-attachments/assets/c0b25de7-bb99-4217-b39a-1f37c901d68d" />
<img width="571" height="351" alt="Screenshot 2025-07-14 at 10 10 03" src="https://github.com/user-attachments/assets/06692c55-43fa-4ce5-8d63-9eb0c55632fe" />
